src/components/tooltip/CallNode.js

```diff
@@ -47,6 +47,13 @@ type Props = {|
   +callTreeSummaryStrategy: CallTreeSummaryStrategy,
 |};
 
+// For debugging purposes, allow tooltips to persist. This aids in inspecting
+// the DOM structure.
+window.persistTooltips = false;
+if (process.env.NODE_ENV === 'development') {
+  console.log('To debug tooltips, set window.persistTooltips to true.');
+}
+
```

Comment:
I believe you meant to remove this code, as you moved it to `window-console.js`.

Reply:
Ah, thanks. I missed this.